packages/driver/package.json

```diff
@@ -3,13 +3,10 @@
   "version": "0.0.0",
   "private": true,
   "scripts": {
-    "check-deps": "node ../../scripts/check-deps.js --verbose",
-    "check-deps-pre": "npm run check-deps -- --prescript",
     "clean-deps": "rm -rf node_modules",
     "cypress:open": "node ../../scripts/cypress open --project ./test",
     "cypress:run": "node ../../scripts/cypress run --project ./test",
-    "prestart": "npm run check-deps-pre",
-    "start": "../coffee/node_modules/.bin/coffee test/support/server.coffee"
+    "start": "$(yarn bin coffee) test/support/server.coffee"
```

Comment:
does this work on windows?

Reply:
Good question, I need to double-check.

Reply:
It seems to work; though I wonder if the options chosen when installing git could impact it working. I tested on Windows 10 Pro, git installed with default options, Powershell (git is also available). If using git bash, then it should work. Using Powershell with default the git shell installed also works.

Reply:
does it work in command prompt? not that i like command prompt, but i think we should support both the shells that users are likely to use on Windows